src/core/qgsdataitem.cpp

```diff
@@ -114,6 +114,31 @@ QVariant QgsFavoritesItem::sortKey() const
   return QStringLiteral( " 0" );
 }
 
+QIcon QgsBookmarksItem::iconBookmarks()
+{
+  return QgsApplication::getThemeIcon( QStringLiteral( "/mActionShowBookmarks.svg" ) );
+}
+
+QVariant QgsBookmarksItem::sortKey() const
+{
+  return QStringLiteral( " 1" );
```

Comment:
Is the space intentional?

Reply:
Yes, to insure the item is placed below favorites.